packages/flutter/lib/src/material/tabs.dart

```diff
@@ -1116,6 +1116,7 @@ class _TabBarViewState extends State<TabBarView> {
   TabController _controller;
   PageController _pageController;
   List<Widget> _children;
```

Comment:
you don't seem to use `_children` at all.

Reply:
This is used for checking if widget got updated at the end of switching logic, in line 1235.

Reply:
yeah that code makes no sense. i hadn't yet commented on it but will now. :-)